includes/upgrades/class-wcs-upgrade-subscription-post-author.php

```diff
@@ -48,7 +48,8 @@ protected function update_item( $subscription_id ) {
 			);
 
 			if ( 0 === $wpdb->rows_affected ) {
-				if ( '1' === get_post_meta( $subscription_id, '_customer_user', true ) && is_a( WCS_Customer_Store::instance(), 'WCS_Customer_Store_Cached_CPT' ) ) {
```

Comment:
These upgrades/ changes are from old upgrade files which will be removed/deprecated in https://github.com/Automattic/woocommerce-subscriptions-core/pull/163.

Not sure if you still want to update these though? I personally don't think it's necessary

Reply:
Right! Reverted this one in https://github.com/Automattic/woocommerce-subscriptions-core/pull/718/commits/d17b8e25ea345d7b30117ed67d0f82e09bfab779